proxima/code/game/items/toys.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/toy/figure/plush/skrell
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "blue plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says: NOT TO USE AS A TOY FOR JOY"
+	icon_state = "axicplushie"
+
+/obj/item/toy/figure/plush/skrell/orange
+	name = "Orange plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says:NOT TO USE AS A TOY FOR JOY"
+	icon_state = "qillplushie"
+
+/obj/item/toy/figure/plush/skrell/red
+	name = "Red plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says:NOT TO USE AS A TOY FOR JOY."
+	icon_state = "xanaplushie"
+
+/obj/item/toy/figure/plush/fox
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "Plush red fox"
+	desc = "Cute and soft plush fox, it's so nice to hug him. After all, he is fluffy like a real fox."
```

Comment:
`him` -> `it`.
`he` - > `it` both toys and foxes have neutral genders unless pets, since they are somehow tend to have female/male genders in the speech. Weirdos.